sacred/experiment.py

```diff
@@ -39,7 +40,8 @@ class Experiment(Ingredient):
     def __init__(self, name: Optional[str] = None,
                  ingredients: Sequence[Ingredient] = (),
                  interactive: bool = False,
-                 base_dir: Optional[PathType] = None):
+                 base_dir: Optional[PathType] = None,
+                 additional_gatherers: Dict[str, Callable] = None):
```

Comment:
I'm not super happy with this name. Any idea for a better name?

Reply:
`additional_host_infos`?